### PR TITLE
(PC-33491) fix(e2e): fix v1.318.1 + refactor bash layer for Robin usage

### PR DESCRIPTION
**Pull request by arenault-pass**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-33491

## Flakiness

If I had to re-run tests in the CI due to flakiness, I add the incident [on Notion][2]

## Checklist

I have:

- [ ] Made sure my feature is working on web.
- [ ] Made sure my feature is working on mobile (depending on relevance : real or virtual devices)
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]
- [ ] I am aware of all the best practices and respected them.

### CODE REVIEW

File: .maestro/tests/subFolder/cloud/commands/SetLocalizationFromEverywhereToParis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+---
+- runFlow:
+    when:
+      visible: 'France entière'
+    commands:
+      - tapOn: 'France entière'
+      - tapOn: 'Choisir une localisation'
+      - inputText: 'Paris'
+      - hideKeyboard
+      - tapOn: 'Paris0Paris'
```
Comment (lbeneston-pass) [inline .maestro/tests/subFolder/cloud/commands/SetLocalizationFromEverywhereToParis.yml:11]: C'est un label `'Paris0Paris'` ?

File: .maestro/tests/subFolder/search/SearchForNoResult.yml [hunk 2]
```diff
@@ -7,15 +7,15 @@ appId: ${MAESTRO_APP_ID}
       visible: 'Ma position'
     commands:
       - tapOn: 'Ma position'
-      - tapOn: 'Partout'
+      - tapOn: 'France entière'
```
Comment (lbeneston-pass) [inline .maestro/tests/subFolder/search/SearchForNoResult.yml:10]: https://www.youtube.com/watch?v=Vtf5_psRzfM&t=3s

**Review by lbeneston-pass — APPROVED** [review]

File: .maestro/tests/subFolder/search/SearchForNoResult.yml (hunk 2 above)
Comment (arenault-pass) [reply .maestro/tests/subFolder/search/SearchForNoResult.yml:10]: bahahah la fameuse vidéo

File: .maestro/tests/subFolder/cloud/commands/SetLocalizationFromEverywhereToParis.yml (hunk 1 above)
Comment (arenault-pass) [reply .maestro/tests/subFolder/cloud/commands/SetLocalizationFromEverywhereToParis.yml:11]: Même pas c'est le wording dans les suggestions de la search, ca ressemble à ça en gros
{ce que tu écris}0{suggestion}

**Pull request merged**